UPGRADE.md

```diff
@@ -54,6 +54,18 @@ class MyException extends \Exception implements ClientAware
 You can always switch to [custom error formatting](https://webonyx.github.io/graphql-php/error-handling/#custom-error-handling-and-formatting)
 to revert to the old format.
 
+### `exit` yourself
+
+The argument `bool $exitWhenDone` was removed from `StandardServer::send500Error()` and `StandardServer::handleRequest()`.
+Exit yourself if you need to:
+
+```php
+$server = new GraphQL\Server\StandardServer();
+-$server->handleRequest($body, true);
++$server->handleRequest($body);
++exit;
+```
```

Comment:
```suggestion
```diff
$server = new GraphQL\Server\StandardServer();
- $server->handleRequest($body, true);
+ $server->handleRequest($body);
+ exit;
```
```

Reply:
nit, the version with spaces before `-` and `+` was better :P

it is the standard way to show diff (to add spaces)
<img width="189" alt="image" src="https://user-images.githubusercontent.com/327717/162200278-bbf4db82-0044-4aed-9fe3-20fa2c18bd79.png">

Reply:
I see, daym